crates/bitwarden/Cargo.toml

```diff
@@ -13,7 +13,11 @@ edition = "2021"
 rust-version = "1.57"
 
 [features]
+default = ["secrets"]
```

Comment:
Is this for backwards compatibility? We should add a deprecation note and update our docs to suggest turning this on.

Reply:
Yeah, it's for backwards compatibility and should be removed eventually, I think. I've updated the bitwarden crate README and the docs to include the feature, we should add a deprecation entry to the changelog once that is merged as well.

Reply:
Added the notice to the changelog